.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: java
 jdk:
-- openjdk7
 - oraclejdk7
```

Comment:
@yanhick @EBatTiVo Perhaps it should be JDK6? If we decide to upload this file somewhere, you know some people use that JDK even now perhaps...